app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -237,12 +237,18 @@ class LuciBuildService {
       maxAttempts: 3,
       delayFactor: Duration(seconds: 2),
     );
+    BatchResponse batchResponse;
     await r.retry(
       () async {
-        await buildBucketClient.batch(BatchRequest(requests: requests));
+        batchResponse = await buildBucketClient.batch(BatchRequest(requests: requests));
```

Comment:
I took a quick look at this and it seems like the problem is that buildBucketClient.batch sometimes takes too long for a regular request(usually 60s timeout). We may need to move the call to this method in something that can run reliably for more than 60s.

Reply:
My expectation is that is already handled, is that correct?

```
// https://cs.opensource.google/flutter/cocoon/+/master:app_dart/lib/src/service/buildbucket.dart;l=83-86
// Null response or statusCode will move to the throw
if (response.statusCode < 300) {
      return responseFromJson(json.decode(rawResponse.substring(kRpcResponseGarbage.length)) as Map<String, dynamic>);
    }
    throw BuildBucketException(response.statusCode, rawResponse);
```

Reply:
Ohh, in that case we need to push the work to a TaskQueue as GitHub sets the 10 second timeout on their request. I created a tracking issue in https://github.com/flutter/flutter/issues/81579